src/sheet.tsx

```diff
@@ -253,18 +253,18 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
                                 {typeof children === 'function'
                                     ? children({closeModal, modalTitleId})
                                     : children}
+                                <div className={styles.modalCloseButton}>
```

Comment:
Moving close button inside the container with the role="dialog" (like it's done in other modals)